src/plugins/partyMode/index.ts

```diff
@@ -0,0 +1,107 @@
+/*
+ * Vencord, a modification for Discord's desktop app
+ * Copyright (c) 2023 Vendicated and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+import { definePluginSettings } from "@api/Settings";
+import { Devs } from "@utils/constants";
+import definePlugin, { OptionType } from "@utils/types";
+import { findStoreLazy } from "@webpack";
+import { GenericStore } from "@webpack/common";
+
+
+const PoggerModeSettingsStore: GenericStore = findStoreLazy("PoggermodeSettingsStore");
+
+enum Intensity {
+    Normal,
+    Better,
+    ProjectX,
+}
+
+const settings = definePluginSettings({
+    superIntensePartyMode: {
+        description: "Party intensity (disable and re-enable plugin to apply changes)",
```

Comment:
Since I added the `onChange` callback, this no longer applies.
```suggestion
        description: "Party intensity",
```